### DISCUSSION TITLE
Support Kotlin Script in Polyglot API

### DISCUSSION THREAD

**Discussion by eutkin** [oracle/graal#3465] [Q&A]
Hi all. Can you advise if there is a way to run kotlin code in a kotlin program via Polyglot API? If it's not possible at the moment, is it planned?

**Comment by thomaswue** [comment]
So you have a Kotlin program that wants to run some other Kotlin program embedded in a new polyglot context?

**Reply by eutkin** [reply]
Yes

**Reply by thomaswue** [reply]
This section in the documentation should provide the information how to do this: https://www.graalvm.org/reference-manual/java-on-truffle/interoperability/#embedding-in-host-java
